### PR TITLE
[pretty] downgrade to clang-format/tidy v9.0

### LINKED ISSUE #5724: [style] llvm@10 brew formula not found
**Describe the bug**
`./script/bootstrap`[ tries to install the](https://github.com/openthread/openthread/blob/master/script/bootstrap#L108) `llvm@10` brew formula, but I don't think it exists. It's not listed in [the formulas available](https://formulae.brew.sh/formula/) in brew's core tap
Screen Shot 2020-10-27 at 1 43 18 PM

**To Reproduce**
Run `./script/bootstrap` on a mac where `clang-format` is not installed

**Expected behavior**
llvm version 10 should be installed

**Console/log output**
```java
$ brew install llvm@10
Error: No available formula or cask with the name "llvm@10".
==> Searching for a previously deleted formula (in the last month)...
Error: No previously deleted formula found.
==> Searching for similarly named formulae...
Error: No similarly named formulae found.
==> Searching taps...
==> Searching taps on GitHub...
Error: No formulae found in taps.
```

### PR DESCRIPTION
**Pull request by jwhui**
Resolves #5724

### CODE REVIEW

**Review by abtink — APPROVED** [review]
👍  Thanks.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -109,7 +109,7 @@ This will open up a text editor where you can specify which commits to squash.
 
 #### Coding Conventions and Style
 
-OpenThread uses and enforces the [OpenThread Coding Conventions and Style](STYLE_GUIDE.md) on all code, except for code located in [third_party](third_party). Use `script/make-pretty` and `script/make-pretty check` to automatically reformat code and check for code-style compliance, respectively. OpenThread currently requires [clang-format v10.0.0](http://releases.llvm.org/download.html#10.0.0) for C/C++ and [yapf v0.29.0](https://github.com/google/yapf) for Python.
+OpenThread uses and enforces the [OpenThread Coding Conventions and Style](STYLE_GUIDE.md) on all code, except for code located in [third_party](third_party). Use `script/make-pretty` and `script/make-pretty check` to automatically reformat code and check for code-style compliance, respectively. OpenThread currently requires [clang-format v9.0.0](http://releases.llvm.org/download.html#10.0.0) for C/C++ and [yapf v0.29.0](https://github.com/google/yapf) for Python.
```
Comment (simonlingoogle) [inline CONTRIBUTING.md:112]:
```suggestion
OpenThread uses and enforces the [OpenThread Coding Conventions and Style](STYLE_GUIDE.md) on all code, except for code located in [third_party](third_party). Use `script/make-pretty` and `script/make-pretty check` to automatically reformat code and check for code-style compliance, respectively. OpenThread currently requires [clang-format v9.0.0](https://releases.llvm.org/download.html#9.0.0) for C/C++ and [yapf v0.29.0](https://github.com/google/yapf) for Python.
```

File: STYLE_GUIDE.md [hunk 2]
```diff
@@ -116,7 +116,7 @@
 
 - OpenThread uses `script/make-pretty` to reformat code and enforce code format and style. `script/make-pretty check` build target is included in OpenThread's continuous integration and must pass before a pull request is merged.
 
-- `script/make-pretty` requires [clang-format v10.0.0](http://releases.llvm.org/download.html#10.0.0) for C/C++ and [yapf v0.29.0](https://github.com/google/yapf) for Python.
+- `script/make-pretty` requires [clang-format v9.0.0](http://releases.llvm.org/download.html#10.0.0) for C/C++ and [yapf v0.29.0](https://github.com/google/yapf) for Python.
```
Comment (simonlingoogle) [inline STYLE_GUIDE.md:119]:
```suggestion
- `script/make-pretty` requires [clang-format v9.0.0](https://releases.llvm.org/download.html#9.0.0) for C/C++ and [yapf v0.29.0](https://github.com/google/yapf) for Python.
```

**Review by wgtdkp — APPROVED** [review]
Thanks! :+1:

**Review by simonlingoogle — APPROVED** [review]
LGTM 👍

**Pull request merged**